Source/Core/DolphinQt/Debugger/NetworkWidget.cpp

```diff
@@ -309,28 +337,98 @@ QGroupBox* NetworkWidget::CreateSSLContextGroup()
   return ssl_context_group;
 }
 
-QGroupBox* NetworkWidget::CreateSSLOptionsGroup()
+QGroupBox* NetworkWidget::CreateDumpOptionsGroup()
 {
-  QGroupBox* ssl_options_group = new QGroupBox(tr("SSL options"));
-  QGridLayout* ssl_options_layout = new QGridLayout;
-  ssl_options_group->setLayout(ssl_options_layout);
-
-  m_dump_ssl_read_checkbox = new QCheckBox(tr("Dump SSL read"));
-  m_dump_ssl_write_checkbox = new QCheckBox(tr("Dump SSL write"));
+  QGroupBox* dump_options_group = new QGroupBox(tr("Dump options"));
```

Comment:
```suggestion
  auto* dump_options_group = new QGroupBox(tr("Dump options"));
```

to avoid repeating yourself when the type name is already written out on the right hand side